docs/en/setup/service-agent/java-agent/Plugin-list.md

```diff
@@ -24,7 +24,7 @@
 - feign-default-http-9.x
 - feign-pathvar-9.x
 - finagle
-- graphql
```

Comment:
I removed this, but it breaks the CI. Maybe there are some other plugin keys should be updated?